py-kms/pykms_WinService.py

```diff
@@ -0,0 +1,55 @@
+# Warning: this script only for Windows 7 SP1 / Windows 2008 R2!
+# -
+# Download links x64:
+# https://www.python.org/ftp/python/3.7.4/python-3.7.4-amd64.exe
+# https://github.com/mhammond/pywin32/releases/download/b224/pywin32-224.win-amd64-py3.7.exe
+# -
+# Download links x32:
+# https://www.python.org/ftp/python/3.7.4/python-3.7.4.exe
+# https://github.com/mhammond/pywin32/releases/download/b224/pywin32-224.win32-py3.7.exe
+# -
+# Need install Python 3.7.4 before pywin32!
+# -
+# 1. Install Python 3.7.4 into C:\Windows\Python374
+# 2. Install pywin32
```

Comment:
New dependencies need to be added to requirements.txt.

Reply:
Nah, it is just the windows service anyways. Nobody will want to install anything for that OS when using the `requirements.txt`. Hopefully.